panel/io/resources.py

```diff
@@ -97,6 +101,15 @@ def conffilter(value):
 
 extension_dirs['panel'] = str(DIST_DIR)
 
+if mimetypes.types_map.get('.js') == 'application/javascript':
```

Comment:
Shouldn't this be `'text/plain'` or `!=`? And maybe only be done on Windows?

Reply:
Yep, it's wrong. Committed code I was testing with.